app/assets/v2/css/town_square.css

```diff
@@ -1004,6 +1004,176 @@ body.green.offer_view .announce {
   }
 }
 
+/* Toggle Switch for Dope Mode */
+input[type=checkbox].toggle-switch {
```

Comment:
we already have switches with bootstrap https://getbootstrap.com/docs/4.4/components/forms/#switches

Reply:
@molecula451 we can remove this now that you're using bootstrap swtiches right. ?

Reply:
Yes, we are using bootstrap switches already

Reply:
> @molecula451 we can remove this now that you're using bootstrap swtiches right. ?

Removed.